roles/matrix-postgres/tasks/setup_postgres.yml

```diff
@@ -162,3 +162,33 @@
     - matrix-change-user-admin-status
     - matrix-postgres-update-user-password-hash
   when: "not matrix_postgres_enabled|bool"
+
+# Create additional databases
+- name: Retrieve IP of postgres container
```

Comment:
Why not use the database server's host name? The Docker DNS server should resolve it for you - {{ matrix_synapse_database_host }}

Reply:
Can you reach a docker container by it's hostname on the host? Last time i checked, a similar feature was only available for docker-on-mac - and ansible (i.e.: this module) runs on the host. The IP however should be routable from the host, regardless of docker version / OS distribution